src/main/java/software/amazon/cloudformation/LambdaWrapper.java

```diff
@@ -200,12 +201,19 @@ public void handleRequest(final InputStream inputStream, final OutputStream outp
             // deserialize incoming payload to modelled request
             try {
                 request = this.serializer.deserialize(input, typeReference);
+
+                handlerResponse = processInvocation(rawInput, request, context);
             } catch (MismatchedInputException e) {
                 JSONObject resourceSchemaJSONObject = provideResourceSchemaJSONObject();
                 JSONObject rawModelObject = rawInput.getJSONObject("requestData").getJSONObject("resourceProperties");
+
                 this.validator.validateObject(rawModelObject, resourceSchemaJSONObject);
+
+                handlerResponse = ProgressEvent.defaultFailureHandler(
+                    new CfnInvalidRequestException("Model validation failed caused by invalid input provided", e),
```

Comment:
This message will be customer visible.  Maybe better wording as "Resource properties validation failed with invalid configuration".